docs/design/libraries/DllImportGenerator/StructMarshalling.md

```diff
@@ -221,6 +210,32 @@ To enable blittable generics support in this struct marshalling model, we extend
 
 Since all fields typed with non-parameterized types are validated to be blittable at type definition time, we know that they are all blittable at type usage time. So, we only need to validate that the generic fields are instantiated with blittable types.
 
+#### Design 2: `DisableRuntimeMarshallingAttribute`
```

Comment:
Maybe also link to approved API proposal.

Reply:
```suggestion
#### Design 2: [`DisableRuntimeMarshallingAttribute`](https://github.com/dotnet/runtime/issues/60639)
```